dev/com.ibm.ws.microprofile.openapi/resources/com/ibm/ws/microprofile/openapi/impl/validation/resources/ValidationMessages.nlsprops

```diff
@@ -39,6 +39,10 @@ headerContentMap=The \"content\" map within header object \"{0}\" must only cont
 
 licenseInvalidURL=The License object must contain a valid URL. The \"{0}\" value specified for the URL is not valid.
 
+linkOperationRefInvalidOrMissing=The Operation object referenced in \"operationRef\" of Link object \"{0}\" is not valid or is missing from the paths.
```

Comment:
Suggestion:
Link object specified a relative \\"operationRef\\" field with value \\"{0}\\" that points to an invalid Operation object.